src/index.js

```diff
@@ -0,0 +1,240 @@
+const state = {
```

Comment:
Great use of state object for temp and color class. You might consider making a separate object to hold the constants